### PR TITLE
conan 0.7.4 (new formula)

### PR DESCRIPTION
**Pull request by lasote**
### All Submissions:

- [ X ] Have you followed the guidelines in our [Contributing](https://github.com/Homebrew/homebrew/blob/master/.github/CONTRIBUTING.md) document?
- [ X ] Have you checked to ensure there aren't other open [Pull Requests](https://github.com/Homebrew/homebrew/pulls) for the same update/change?

_You can erase any parts of this template not applicable to your Pull Request._

### New Formulae Submissions:

- [ X ] Does your submission pass
`brew audit --strict --online <formula>` (where `<formula>` is the name of the formula you're submitting)?
- [ X ] Have you built your formula locally prior to submission with `brew install <formula>`?

### CODE REVIEW

File: Library/Formula/conan.rb [hunk 1]
```diff
@@ -0,0 +1,72 @@
...
+
+    ENV.prepend_create_path "PYTHONPATH", libexec/"lib/python2.7/site-packages"
+    system "python", *Language::Python.setup_install_args(libexec)
+
+    bin.install Dir[libexec/"bin/*"]
+    bin.env_script_all_files(libexec/"bin", :PYTHONPATH => ENV["PYTHONPATH"])
+  end
+
+  test do
+    system "#{bin}/conan", "--version"
```
Comment (mikemcquaid) [inline Library/Formula/conan.rb]: Can this test be modified to do something more substantial than e.g. `--version` or `--help`? See `cmake.rb` for an example of an application formula with a good test and `tinyxml2.rb` for an example of a library formula with a good test. Thanks!

Comment (mikemcquaid) [inline Library/Formula/conan.rb]: i.e. install a package

Comment (lasote) [inline Library/Formula/conan.rb]: Updated PR

File: Library/Formula/conan.rb [hunk 2]
```diff
@@ -0,0 +1,72 @@
+class Conan < Formula
+  desc "This tool is a distributed, open source, package manager for C/C++"
```
Comment (dunn) [inline Library/Formula/conan.rb]: This can be simplified to just "Distributed, open source, package manager for C/C++"

Comment (lasote) [inline Library/Formula/conan.rb]: Ok. Updated PR. thanks

**Comment by mikemcquaid** [comment]
Thanks for your contribution to Homebrew! Without people like you submitting PRs we couldn't run this project. You rock!

**Pull request closed without merging**